coalib/bears/Bear.py

```diff
@@ -160,6 +177,22 @@ def include_local_files(cls):
         return ((cls.INCLUDE_LOCAL_FILES,) if isinstance(
             cls.INCLUDE_LOCAL_FILES, str) else cls.INCLUDE_LOCAL_FILES)
 
+    @classproperty
+    def can_detect(cls):
+        """
+        :return: A tuple with CAN_DETECTs.
+        """
+        return ((cls.CAN_DETECT,) if isinstance(
+            cls.CAN_DETECT, str) else cls.CAN_DETECT)
```

Comment:
while we're at it: if we switch to sets we can drop the properties as theres no confusion about any lonely comma, you always use a set.